manatools/services.py

```diff
@@ -142,16 +150,20 @@ def set_service(self, service, enable):
         legacy = os.path.isfile("/etc/rc.d/init.d/{}".format(service))
         if service in self._xinetd_services.keys():
             env = {'LANGUAGE': 'C', 'PATH': "/usr/bin:/usr/sbin"}
-            # TODO : Change to force root command
-            try:
-                chkconf = subprocess.run(['/usr/sbin/chkconfig', "-add" if enable else "--del",
-                                          service], env=env, timeout=120, check=True, capture_output=True, text=True)
-            except (subprocess.CalledProcessError, subprocess.TimeoutExpired):
-                # TODO return an exception to the exterior
-                print("chkconfig error when trying to add/delete service", stderr)
+#            if dbus_context.is_authorized('org.freeedesktop.policykit.exec', {'polkit.icon': 'abcd', 'aaaa': 'zzzz'}, interactive=True):
+            #     try:
+            #         chkconf = subprocess.run(['/usr/sbin/chkconfig', "-add" if enable else "--del",
+            #                                   service], env=env, timeout=120, check=True, capture_output=True, text=True)
+            #     except (subprocess.CalledProcessError, subprocess.TimeoutExpired):
+            #         # TODO return an exception to the exterior
+            #         print("chkconfig error when trying to add/delete service", stderr)
+            # else:
+            #     # TODO return an excpetion to the exterior
+            #     print("You are not authorized to perform this action", stderr)
```

Comment:
those comments are needed to recall something or have you forgot to remove them?